packages/sign-client/src/controllers/engine.ts

```diff
@@ -878,6 +891,43 @@ export class Engine extends IEngine {
     }
   };
 
+  private addRequestToQueue = (request: PendingRequestTypes.Struct) => {
+    this.requestQueue.requests.push(request);
+  };
+
+  private afterResponse = (params: EngineTypes.RespondParams) => {
```

Comment:
Would try to keep naming conventions in line with `<verb>...`

```suggestion
  private cleanupAfterResponse = (params: EngineTypes.RespondParams) => {
```